templates/incl-editor-and-output.html

```diff
@@ -133,6 +133,12 @@
             </div>
           </form>
         </div>
+        <div class="pygame-modal py-2 text-left px-3 border-4 border-teal-500 mt-3 rounded bg-green-100" style="display: none">
```

Comment:
Same here, please use `id='pygame-modal'` instead.

Reply:
Sorry! forgot about that convention, [Change class to id](https://github.com/hedyorg/hedy/pull/3668/commits/5da051dc60a3d17db45a904f07156b8fa4074125) fixes this.